homeassistant/components/fujitsu_hvac/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Enter your FGLair credentials",
+        "data": {
+          "is_europe": "Use european servers",
```

Comment:
When does the user need to select this? Why should the user select this?

Reply:
The API uses different serrvers when the user is in europe vs elsewhere (I'm guessing because of GPDR issues).

Reply:
Maybe add an explanation in the data description field of the field